apps/task/migrations/0001_initial.py

```diff
@@ -1,35 +1,59 @@
 
-from django.conf import settings
-from django.db import migrations, models
+import uuid
+
 import django.db.models.deletion
 import django.utils.timezone
-import uuid
+from django.conf import settings
+from django.db import migrations, models
 
 
 class Migration(migrations.Migration):
-
     initial = True
 
     dependencies = [
         migrations.swappable_dependency(settings.AUTH_USER_MODEL),
-        ('river', '0006_alter_river_current_stage'),
+        ("river", "0006_alter_river_current_stage"),
     ]
 
     operations = [
         migrations.CreateModel(
-            name='Task',
+            name="Task",
             fields=[
-                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('created_at', models.DateTimeField(default=django.utils.timezone.now)),
-                ('uuid', models.UUIDField(default=uuid.uuid4, editable=False)),
-                ('name', models.CharField(max_length=120)),
-                ('stage_name', models.CharField(choices=[('plan', 'Plan'), ('act', 'Act')], max_length=8)),
-                ('topic', models.CharField(max_length=32)),
-                ('done', models.BooleanField(default=False)),
-                ('due', models.DateTimeField(blank=True, null=True)),
-                ('responsible', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
-                ('river', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='river.river')),
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created_at", models.DateTimeField(default=django.utils.timezone.now)),
+                ("uuid", models.UUIDField(default=uuid.uuid4, editable=False)),
+                ("name", models.CharField(max_length=120)),
+                (
+                    "stage_name",
+                    models.CharField(
+                        choices=[("plan", "Plan"), ("act", "Act")], max_length=8
+                    ),
+                ),
+                ("topic", models.CharField(max_length=32)),
+                ("done", models.BooleanField(default=False)),
+                ("due", models.DateTimeField(blank=True, null=True)),
+                (
+                    "responsible",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to=settings.AUTH_USER_MODEL,
+                    ),
+                ),
+                (
+                    "river",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="river.river"
+                    ),
+                ),
```

Comment:
I think this file got `black`-formatted right? I tend to exclude the migrations dir from formatting tools, considering them as generated files, but alternatively could _always_ format them.

Reply:
I think django is producing them black-formatted [since 4.1](https://docs.djangoproject.com/en/4.2/releases/4.1/#management-commands), but it seems this migration was generated before we update to 4.2: https://github.com/animorphcoop/shared-futures-space/blob/877ab3f938dde9d0cc1b2b4ea9f9f255c505c8fd/apps/task/migrations/0001_initial.py#L1